integr8ly/integreatly-operator
# MGDAPI-4551 reduce the error budget and slo % dashboards to 7d

# Issue link
https://issues.redhat.com/browse/MGDAPI-4551

# What
Reduce the Error Budget and SLO % dashboard to 7 days to align with weekly SRE / program updates

# Verification steps
- Run the integreatly-operator locally
- Check that the following dashboards are updated from 28d to 7d:
  - Critical SLO summary
  - SLO SSO Availability - 5xx HAProxy Errors
- Additional verifications will have to be carried out on eng-long-lived once the changes go live and run for a few days ([JIRA](https://issues.redhat.com/browse/MGDAPI-4762))
